code/game/turfs/simulated.dm

```diff
@@ -23,7 +23,8 @@
  		overlays += wet_overlay
  		sleep(800)
  		if(wet >= 2)
- 			return
+ 			sleep(1600)
+ 			wet = 0

```

Comment:
Unnecessary operation. Could just have the sleep command, it'll hold the proc and run into the `wet = 0` assignment on the next line.